learn/core_concepts/documents.mdx

```diff
@@ -6,6 +6,10 @@ sidebarDepth: 3
 
 A document is an object composed of one or more fields. Each field consists of an **attribute** and its associated **value**. Documents function as containers for organizing data, and are the basic building blocks of a Meilisearch database. To search for a document, you must first add it to an [index](/learn/core_concepts/indexes).
 
+<Capsule intent="note">
+Meilisearch orders documents in the order they were inserted into the database.
+</Capsule>
+
```

Comment:
This feels a bit random. This page defines a document and gives pointers on its constituent parts and etc. Why mention this here? I'm not convinced this is an improvement over leaving it in the API reference and am afraid some might confuse this ordering with relevancy ranking, or even sorting.

How can people use this information? As in, when is it relevant for a user to know the internal document order? I think answering these questions is very important when deciding the best location for this tidbit of data.

Reply:
I think it fits under the "add or update documents" and "add or replace document" endpoints